### PR TITLE
Add Another Applicant

### PR DESCRIPTION
**Pull request by anuroop15**
Pre-int issue

### CODE REVIEW

File: app/javascript/forms/cardsMain.jsx [hunk 1]
```diff
@@ -7,10 +7,14 @@ import Employment from './employment_card'
 export default class Cards extends React.Component {
   constructor (props) {
     super(props)
+    this.clickClose = this.clickClose.bind(this)
     this.state = {
       isFocused: {}
     }
   }
+  clickClose (id) {
```
Comment (ratneshraval) [inline app/javascript/forms/cardsMain.jsx]: You'll probably have to rename this to properly reflect that this is for applicant card close. When we lift the state up, other close methods will probably end up here.

File: app/javascript/forms/cardsMain.jsx [hunk 2]
```diff
@@ -22,30 +26,32 @@ export default class Cards extends React.Component {
   }
   render () {
     const {formData} = this.state
+    let showClose = (this.props.id === 0) ? false : true
     return (
       <div className='cards'>
-        <div id="nameSection" onClick={this.toggleOnFocus.bind(this, 'nameSection')} className={(this.state.isFocused['nameSection']) + ' ' + 'card name-section double-gap-top'}>
+        {showClose && <span onClick={() => this.clickClose(this.props.id)} className='pull-right glyphicon glyphicon-remove' />}
```
Comment (ratneshraval) [inline app/javascript/forms/cardsMain.jsx]: I think this syntax is experimental and we should stick to bind method.

**Comment by ratneshraval** [comment]
@lauherkcambria 
2. There is an X button to delete applicant card

**Comment by lauherkcambria** [comment]
ready to go!

**Pull request merged**